apps/chat-e2e/src/assertions/folderAssertion.ts

```diff
@@ -364,4 +364,51 @@ export class FolderAssertion {
           .soft(folderLocator, ExpectedMessages.folderIsNotVisible)
           .toBeHidden();
   }
+
+  public async assertFolderArrowIconState(
+    folder: TreeEntity,
+    expectedState: ElementState,
+  ) {
+    const arrowIcon = this.folder.getFolderArrowIcon(folder.name, folder.index);
+    expectedState === 'visible'
+      ? await expect
+          .soft(arrowIcon, ExpectedMessages.sharedEntityIconIsVisible)
+          .toBeVisible()
+      : await expect
+          .soft(arrowIcon, ExpectedMessages.sharedEntityIconIsNotVisible)
+          .toBeHidden();
+  }
+
+  public async assertFolderArrowIconColor(
```

Comment:
Are we talking here about shared folder arrow only or about any arrow? Perhaps you should name this method assertSharedFolderArrowIconColor?

Reply:
agree, done